File: Java/TP1-2/src/AdivinheMeuNumero.java
import java.util.Scanner;

public class AdivinheMeuNumero
{
  public static void main(String[] args)
  {
  
    AdvinhatorClass adivinha = new AdvinhatorClass();
    
    boolean continua = false;
    
    while (!continua) {
        System.out.println("Entre com o seu número: ");
       	Scanner leitor = new Scanner(System.in);		
      	adivinha.advinhation(leitor.nextInt());
      	
    }
    
  }  
}

File: Kotlin/KTP1-2/src/AdivinheMeuNumero.kt
import java.util.Scanner;

object AdivinheMeuNumero {
	@JvmStatic fun main(args: Array<String>) {
		val adivinha = AdvinhatorClass()
		val continua = false
		while (!continua) {
			println("Entre com o seu numero: ")
			val leitor = Scanner(System.`in`)
			adivinha.advinhation(leitor.nextInt())
		}
	}
}

